tensor.go

```diff
@@ -320,8 +320,8 @@ func Add(a, other Tensor, alpha float32) Tensor {
 	return Tensor{(*unsafe.Pointer)(&t)}
 }
 
-// AddInplace torch.add_
-func (a *Tensor) AddInplace(other Tensor) Tensor {
+// Add adds in-place
+func (a *Tensor) Add(other Tensor) Tensor {
```

Comment:
According to the discussion, we use `AddI` to represent the Add inplace.

Then we will have:

```
func (t Tensor) AddI(Tensor) Tensor
func (t Tensor) Add(Tensor) Tensor
func Add(a, b Tensor) Tensor
```